### PR TITLE
Cameron Bacon

### PR DESCRIPTION
**Pull request by cameronmbacon**

### CODE REVIEW

File: lab-cameron/routes/pokemon-routes.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    if (!req.params.id) return res.status(400).send(createError(400, 'bad request'));
+    pokemonCtrl.fetchPokemon(req.params.id)
+    .then(pokemon => {
+      console.log(pokemon);
+      res.json(pokemon);
+    })
+    .catch(err => res.status(400).send(err.message));
+  });
+  router.get('/api/pokemon', (req, res) => {
+    pokemonCtrl.gottaFetchEmAll()
```
Comment (ghost) [inline lab-cameron/routes/pokemon-routes.js:17]: gotta fetch 'em all LMAO

File: lab-cameron/routes/pokemon-routes.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+'use strict';
```
Comment (JudyVue) [inline lab-cameron/routes/pokemon-routes.js:1]: This module looks good, but for a first glance, might be good to space things out between your routes, i.e. indent a line between ``router.get`` and ``router.post``.

File: lab-cameron/routes/pokemon-routes.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  router.get('/api/pokemon/:id', (req, res) => {
+    if (!req.params.id) return res.status(400).send(createError(400, 'bad request'));
+    pokemonCtrl.fetchPokemon(req.params.id)
+    .then(pokemon => {
+      console.log(pokemon);
+      res.json(pokemon);
+    })
+    .catch(err => res.status(400).send(err.message));
+  });
+  router.get('/api/pokemon', (req, res) => {
```
Comment (JudyVue) [inline lab-cameron/routes/pokemon-routes.js:16]: Good job doing the bonus and great controller name!

File: lab-cameron/routes/pokemon-routes.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    pokemonCtrl.gottaFetchEmAll()
+    .then(pokemon => {
+      console.log(pokemon);
```
Comment (JudyVue) [inline lab-cameron/routes/pokemon-routes.js:19]: **TODO FOR LOST POINTS**

Take out stray console.logs before submission, or manage them on another branch and submit a clean branch to me.

File: lab-cameron/test/server-test.js [hunk 5]
```diff
@@ -0,0 +1,221 @@
+'use strict';
```
Comment (JudyVue) [inline lab-cameron/test/server-test.js:1]: Tests are great!

File: lab-cameron/README.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+![cf](https://i.imgur.com/7v5ASc8.png) mongo-express-pokémon-API
+======
+
```
Comment (JudyVue) [inline lab-cameron/README.md:3]: Good REAMDE.

**Pull request merged**